testapp/models.py

```diff
@@ -15,3 +17,19 @@ class Case(models.Model):
         blank=True,
         null=True,
     )
+
+
+class ZgwConfig(models.Model):
+    ztc_service = models.OneToOneField(
```

Comment:
I don't see the relevance of this model field for this patch at all - please explain why it's needed and otherwise remove it.